code/modules/mob/living/emote.dm

```diff
@@ -627,4 +627,7 @@
 	message_ipc = "whistles a few synthesized notes"
 
 /datum/emote/living/whistle/get_sound(mob/living/user)
-	return 'sound/items/megaphone.ogg'
+	if(MOB_ROBOTIC in user.mob_biotypes)
+		return 'sound/items/megaphone.ogg'
+	else
+		return 'sound/magic/warpwhistle.ogg'
```

Comment:
wouldn't this reduce the "tell" of the wizard's warp whistle as now anyone can perform the emote?

Reply:
you could say the same of the megaphone sfx

Reply:
The megaphone is a mundane tool found in any department and means very little, the warp whistle indicates that a wizard is nearby, which is much, much more important than a megaphone. While you could "say the same" they're different magnitudes of importance

(also you could probably find a much much better sound for this on freesound or smth)